.travis.yml

```
@@ -17,6 +17,7 @@ services:
 script:
 
   - docker run --name selenium-container -d --net host -v /dev/shm:/dev/shm selenium/standalone-chrome
+  - make lint
```

I think this will fix your failing build
```suggestion
  # Black formatter only works on Python 3.6+
  - if [[ $TRAVIS_PYTHON_VERSION > '3.59' ]]; then make lint; fi
```